libraries/mpxe-store/src/store.c

```diff
@@ -14,7 +14,9 @@
 #include "log.h"
 #include "stores.pb-c.h"
 
-#define MAX_STORE_COUNT 64
+#define MAX_STORE_COUNT 257
```

Comment:
Not super impressed you wanted to go about it this way, what about stores beyond just pca9539r and gpio? not a super great change

Reply:
Yeah, I just added the total for now and I figured we could increase the maximum store number as we need it.
I just needed 257 for my testing of pca9539r. I could change it to a 1000.